chia/full_node/full_node_api.py

```diff
@@ -1256,11 +1256,10 @@ async def request_removals(self, request: wallet_protocol.RequestRemovals) -> Op
         return msg
 
     @api_request
-    async def send_transaction(self, request: wallet_protocol.SendTransaction) -> Optional[Message]:
+    async def send_transaction(self, request: wallet_protocol.SendTransaction, test=False) -> Optional[Message]:
```

Comment:
```suggestion
    async def send_transaction(self, request: wallet_protocol.SendTransaction, *, test=False) -> Optional[Message]:
```

Reply:
It's nice to force these named arguments to be named, especially boolean arguments.